### PR TITLE
Signup email check

### PR DESCRIPTION
**Pull request by yajo**
This module was written to extend the functionality of public sign up to force
users to provide a valid email address.

To reinforce this requirement, the user will not need to provide a password at
sign up, but when logging in later for the first time.

To use this module, you need to:
- Log out.
- Sign up `/web/signup` with a valid email.

CC @rafaelbn @sergio-incaser.

### CODE REVIEW

**Comment by rafaelbn** [comment]
@Yajo must fix check failing in runbot first

**Comment by dreispt** [comment]
@Yajo did you find troublesome the harcoded 1 day validity for the signup token?

**Comment by rafaelbn** [comment]
@Yajo after signup Odoo says " Check your email to activate your account! " but there is not any message sent. We expect that Odoo sends an email with the link for confirming it.

**Comment by sergio-teruel** [comment]
@Yajo I test into a private server with a valid email setting. when are you send to user the validation email?

**Comment by yajo** [comment]
Thanks everyone, I'll send some commits but please do not review until I say.

**Comment by yajo** [comment]
It should be ready for prime time. Travis should work too. However, runbot will fail because of new dependency on `validate_email` python package.

**Comment by yajo** [comment]
@dreispt That does not matter.

**Comment by rafaelbn** [comment]
Tested :+1: , please @sergio-incaser @carlos-incaser could you review again.

**Comment by rafaelbn** [comment]
@pedrobaeza @dreispt how to solve this

> It should be ready for prime time. Travis should work too. However, runbot will fail because of new dependency on validate_email python package

**Comment by StefanRijnhart** [comment]
Let's ask @gurneyalex to add the validate_email python library to the runbot configuration.

File: auth_signup_verify_email/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
+        "python": [
+            "validate_email",
+        ],
+    },
+    "depends": [
+        "auth_signup",
```
Comment (StefanRijnhart) [inline auth_signup_verify_email/__openerp__.py:20]: If you don't depend on the _website_ module, then what is your reason to add this module here?

**Comment by sergio-teruel** [comment]
@Yajo @rafaelbn Tested in local server and all ok, for me :+1:

File: auth_signup_verify_email/__openerp__.py (hunk 1 above)
Comment (yajo) [inline auth_signup_verify_email/__openerp__.py:20]: Good question... Where should it go?

Comment (yajo) [inline auth_signup_verify_email/__openerp__.py:20]: Ah nevermind, found it: https://github.com/OCA/server-tools. Let me close this and resend there.

**Comment by yajo** [comment]
Closing PR to open in server-tools. Sorry for that.

File: auth_signup_verify_email/__openerp__.py (hunk 1 above)
Comment (StefanRijnhart) [inline auth_signup_verify_email/__openerp__.py:20]: Thanks!

**Pull request closed without merging**